scalafix-core/shared/src/main/scala/org/langmeta/internal/ScalafixLangmetaHacks.scala

```diff
@@ -11,8 +11,10 @@ object ScalafixLangmetaHacks {
       message: String): String = {
     if (pos != Position.None) {
       val input = pos.input
+      val startLine = pos.startLine + 1
+      val startColumn = pos.startColumn + 1
       val header =
-        s"${input.syntax}:${pos.startLine + 1}: $severity: $message"
+        s"${input.syntax}:$startLine:$startColumn $severity: $message"
```

Comment:
you still need another `:` after the `startColumn`.

```
s"${input.syntax}:$startLine:$startColumn: $severity: $message"
```

Reply:
and AFAICT scalac doesn't use `severity`

Reply:
Fixed.

 scalac includes the severity before the message

```
foo.scala:2: error: type mismatch;
 found   : Int(1)
 required: String
  val x: String = 1
                  ^
```